frontend/src/views/CanteenEditor/PublicationForm/index.vue

```diff
@@ -266,6 +281,13 @@ export default {
         },
       }[this.isPublished]
     },
```

Comment:
J'ai une suggestion pour ces méthodes qui pourrait simplifier les choses. On aurait simplement une valeur qui dit combien d'images sont dans le header, et une autre qui rend les "autres" images. Donc : 
1- Pas besoin de la variable `idealImageMax`, car elle ne serait utilisé que dans `imageHeaderLimit`
2- Selon moi, en vue mobile, on ne montre aucune image dans le header, elles seraient toutes en bas. Ça fait bizarre de devoir carrément scroller pour trouver le nom de la cantine.

```javascript
    // idealImageMax: 3 Plus besoin

    imageHeaderLimit() {
      return this.$vuetify.breakpoint.xs ? 0 : 3
    },
    additionalImagesMax() {
      return Math.max(this.canteen.images.length, this.imageHeaderLimit)
    },
```

Reply:
> 2- Selon moi, en vue mobile, on ne montre aucune image dans le header, elles seraient toutes en bas. Ça fait bizarre de devoir carrément scroller pour trouver le nom de la cantine.

pas contre en soi mais normalement pour le vue mobile le nom de la cantine est tout en haut, et la première image à la suite. Tu veux toujours modifier pour mettre tout en bas ?

Reply:
Parlons en vive voix de ta suggestion de refacto - dit moi quand t'es dispo

Reply:
Car ici sur xs le `additionalImagesLimit` est 0 si l'utilisateur n'a pas saisi des images, et alors telquel l'utilisateur ne pourrait pas ajouter des images quand iel utilise vue mobile

Reply:
En effet, on pourrait avoir un premier step avec une image en haut, puis itérer si besoin. Ce serait genre : 

```javascript
    imageHeaderLimit() {
      return this.$vuetify.breakpoint.xs ? 1 : 3
    },
    additionalImagesMax() {
      return Math.max(this.canteen.images.length, this.imageHeaderLimit)
    },
```

Reply:
J'etais un idiot, je comprends maintenant les suggestions, c'est fait